main.swift

```diff
@@ -361,9 +361,22 @@ do {
     }
     
     try performCommand(description: "Copying template folder") {
+        let ignorableItems = ["readme.md", "license"]
+        let ignoredItems = try fileManager.contentsOfDirectory(atPath: destination).map {
```

Comment:
My initial implementation was:

~~~swift
if ignorableItems.contains(itemName) {
    guard fileManager.fileExists(atPath: destinationPath) == false else {
        continue
    }
}
~~~

But it has 2 problems IMHO:

- not the best syntax (more indents)
- case sensitive (`Readme.md` won't be matched for instance)
 
Let me know if you think this implementation's better 🙂